test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Your test.md file should be ignored by .gitignore file as required in the requirements. Please ignore this file by adding  it to .gitignore file :+1:

Reply:
added test.md file on the gitignore